mlir/lib/Target/Cpp/TranslateToCpp.cpp

```diff
@@ -1353,16 +1364,22 @@ bool CppEmitter::hasBlockLabel(Block &block) {
 }
 
 LogicalResult CppEmitter::emitAttribute(Location loc, Attribute attr) {
+  return CppEmitter::emitAttributeToArbitraryStream(loc, attr, os);
+}
+
+LogicalResult CppEmitter::emitAttributeToArbitraryStream(Location loc,
```

Comment:
nit: As matter of taste, I would remove `Arbitrary` and call this `emitAttributeToStream`.